packages/simple-markdown/flow-typed/npm/express_v4.17.x.js

```diff
@@ -0,0 +1,353 @@
+// flow-typed signature: 53fbc99af5ac2f0732fc7c8641f3e128
```

Comment:
I wonder if we still need express?  I see `babel-mark.js` but no idea what that does.